.eslintignore

```diff
@@ -1 +1,2 @@
 node_modules
+lib
```

Comment:
I will reset the last commit to remove the changes on `.eslintignore` and `.gitignore`, they are only for the ephemeral test environment purposes.